electron/renderer/src/components/WebView/Webview.tsx

```diff
@@ -226,30 +256,36 @@ const Webview = ({
       }
     };
     const ON_IPC_MESSAGE = 'ipc-message';
-    webviewRef.current.addEventListener(ON_IPC_MESSAGE, onIpcMessage);
+    webviewRef.current?.addEventListener(ON_IPC_MESSAGE, onIpcMessage);
     return () => {
       if (webviewRef.current) {
-        webviewRef.current.removeEventListener(ON_IPC_MESSAGE, onIpcMessage);
+        webviewRef.current?.removeEventListener(ON_IPC_MESSAGE, onIpcMessage);
       }
     };
   }, [account, accountLifecycle, conversationJoinData]);
 
-  const deleteWebview = account => {
-    window.sendDeleteAccount(account.id, account.sessionID).then(() => {
+  const deleteWebview = async (account: Account) => {
+    await window.sendDeleteAccount(account.id, account.sessionID).then(() => {
       abortAccountCreation(account.id);
     });
   };
 
   return (
     <>
       <LoadingSpinner visible={!!account.visible} webviewRef={webviewRef} />
+
       <webview
-        allowpopups="true"
+        // eslint-disable-next-line react/no-unknown-property
+        allowpopups
         className={`Webview${account.visible ? '' : ' hide'}`}
         data-accountid={account.id}
+        // @ts-ignore
+        // eslint-disable-next-line react/no-unknown-property
         visible={String(!!account.visible)}
         src={url}
+        // eslint-disable-next-line react/no-unknown-property
         partition={account.sessionID ? `persist:${account.sessionID}` : ''}
+        // eslint-disable-next-line react/no-unknown-property
```

Comment:
All those disabling make me a little curious, why do we have those properties if they are unknown? Let's try to understand what is happening here and migrate/remove if needed